client/components/RatingsReviews/handleReviewClicks.jsx

```diff
@@ -22,4 +22,16 @@ export const reportClick = (e) => {
   //   .catch((err) => {
   //     console.log(err);
   //   });
+};
+
+export const submitReview = (review, setReviews, reviews) => {
+  let url = 'proxy/api/fec2/hratx/reviews/';
+  review = JSON.parse(JSON.stringify(review));
```

Comment:
just, why

Reply:
oh right it was one of my desperate bug fixing attempts. sorry for any bleeding eyeballs